cmake/modules/Findpmem.cmake

```diff
@@ -16,17 +16,10 @@ foreach(component ${pmem_FIND_COMPONENTS})
   if(found EQUAL -1)
     message(FATAL_ERROR "unknown libpmem component: ${component}")
   endif()
-  set(pkg_module_spec "lib${component}")
-  if(pmem_FIND_VERSION_EXACT)
-    set(pkg_module_spec "lib${component}=${pmem_FIND_VERSION}")
-  elseif(pmem_FIND_VERSION)
-    set(pkg_module_spec "lib${component}>=${pmem_FIND_VERSION}")
-  endif()
-  pkg_check_modules(PKG_${component} QUIET ${pkg_module_spec})
+  pkg_check_modules(PKG_${component} QUIET "lib${component}")
```

Comment:
Per my understanding, now `pmem_VERSION_STRING` will always be set to the minimun version of the found components. So the version check in `find_package_handle_standard_args()` can fail correctly if `pmem_VERSION_STRING` doesn't satisfy the user-specified version.